harness/determined/cli/trial.py

```
@@ -121,6 +126,69 @@ def trial_logs(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def generate_support_bundle(args: Namespace) -> None:
+    output_dir = args.output_dir
+    if output_dir is None:
+        output_dir = os.getcwd()
+
+    elif not os.path.exists(output_dir):
+        os.mkdir(output_dir)
```

I would probably just raise a value error here instead.